Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -618,7 +627,45 @@ class CreateSafeFormUIModel {
         })
     }
 
+    func relaySubmit() {
+        relayingTask?.cancel()
+        relayingTask = relay(completion: { [weak self] result in
+            guard let self = self else { return }
+
+            switch result {
+            case .failure(let error):
+                self.didSubmitFailed(error)
+            case .success:
+                Tracker.trackEvent(.relayUserSuccess)
+                self.didSubmitSuccess()
+            }
+        })
+    }
+
+     func relay(completion: @escaping (Result<Void, Error>) -> Void) -> URLSessionTask? {
+        guard let tx = transaction else { return nil }
+        let task = relayerService.asyncRelayTransaction(chainId: chain!.id!,
+                                                        to: Address(tx.to),
+                                                        txData: tx.data.storage.toHexStringWithPrefix()
+        ) { [weak self] response in
+            guard let self = self else { return }
+            switch(response) {
+            case .success:
+                DispatchQueue.main.async {
+                    self.didSubmitTransaction(txHash: Eth.Hash(tx.txHash().storage))
+                    completion(.success(()))
+                }
+            case .failure(let error):
+                dispatchOnMainThread(completion(.failure(error)))
+            }
+        }
+        return task
+    }
+
+    // this creates a tx and sends it to infura for execution
     func send(completion: @escaping (Result<Void, Error>) -> Void) -> URLSessionTask? {
+
+        Tracker.trackEvent(.relayUserExecTxPaymentSigner)
```

Comment:
Maybe for future pr, but relayUserExecTxPaymentSigner needs to be renamed and drop relay at the beginning